composer.json

```diff
@@ -6,16 +6,15 @@
         "sort-packages": true
     },
     "require": {
-        "php": "^5.6|^7.0",
+        "php": "^5.6 || ^7.0 || ^8.0",
```

Comment:
Probably time to drop support for unsupported versions. As a separate PR, of course.